components/template-mgt/org.wso2.carbon.identity.template.mgt/src/test/java/org/wso2/carbon/identity/template/mgt/TemplateManagerImplTest.java

```diff
@@ -53,6 +53,7 @@
 import static org.wso2.carbon.identity.template.mgt.util.TestUtils.mockDataSource;
 import static org.wso2.carbon.identity.template.mgt.util.TestUtils.spyConnection;
 
+@Deprecated
```

Comment:
this is also a test class of a core class.. let's keep it

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2996/commits/1aa997d374f2254338f5287b2bc2fef815104ff3